integration/spark/shared/src/testScala212/java/io/openlineage/spark/agent/lifecycle/LogicalPlanSerializer212Test.java

```diff
@@ -72,7 +76,10 @@ public boolean canEqual(Object that) {
     final ObjectMapper mapper = new ObjectMapper();
     try {
       Logger.getLogger(logicalPlanSerializer.getClass()).setLevel(Level.ERROR);
-      mapper.readTree(logicalPlanSerializer.serialize(notSerializablePlan));
+      String serialize = logicalPlanSerializer.serialize(notSerializablePlan);
+      JsonNode jsonNode = mapper.readTree(serialize);
```

Comment:
This test used to return MissingNode object. It is not a valid JSON in my opinion.